test/metabase/driver/bigquery_test.clj

```diff
@@ -3,12 +3,15 @@
             [metabase
              [query-processor :as qp]
              [query-processor-test :as qptest]]
-            [metabase.query-processor.expand :as ql]
+            [metabase.query-processor.middleware.expand :as ql]
             [metabase.test
              [data :as data]
              [util :as tu]]
             [metabase.test.data.datasets :refer [expect-with-engine]]))
 
+(def col-defaults
```

Comment:
same thing, if we make this `^:private` it will make our lives easier should we choose to refactor it in the future